packages/@webex/plugin-meetings/src/member/index.ts

```diff
@@ -220,6 +221,16 @@ export default class Member {
      * @memberof Member
      */
     this.isModeratorAssignmentProhibited = null;
+    /**
+     * @instance
+     * @type {Object}
+     * @public
+     * @memberof Member
+     */
+    this.mediaStatus = {
```

Comment:
@marcin-bazyl  do you like the name? , and the values will be sendonly , sendrecv etc which I feel is ok for developers any thoughts

Reply:
values will be:
RECVONLY, SENDONLY, SENDRECV, INACTIVE, UNKNOWN

Reply:
Can you add a type for it and example in the comment section

Reply:
Done

Reply:
> @marcin-bazyl do you like the name?

yeah the name is fine